Android/res/layout/popup_info_gps.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    style="?android:attr/buttonBarStyle"
+    android:paddingStart="5dp"
+    android:paddingEnd="5dp"
+    >
+
+    <TextView
+        android:id="@+id/bar_gps_satno"
+        style="?android:attr/buttonBarButtonStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="left|center_vertical"
+        android:textColor="@android:color/black"
+        android:text="Satellites 3"
```

Comment:
@funtastix the default `android:text` value should be the empty content string: [`@string/empty_content`](https://github.com/DroidPlanner/droidplanner/blob/9605649e4269c363819d7c01e069d9061f7a0141/Android/res/values/strings.xml#L453)